src/python/build_management/build_manager.py

```diff
@@ -663,38 +663,24 @@ def _setup_application_path(self,
       os.system('rm %s' % symbolic_link_target)
       os.system('ln -s %s %s' % (app_directory, symbolic_link_target))
 
-    # Android specific initialization.
-    if environment.platform() == 'ANDROID':
-      # Prepare device for app install.
-      android.device.initialize_device()
-
-      # On Android, we may need to write a command line file. We do this in
-      # advance so that we do not have to write this to the device multiple
-      # times.
-      # TODO(mbarbella): Build code should not depend on fuzzing.
-      from fuzzing import testcase_manager
-      testcase_manager.get_command_line_for_application(
-          write_command_line_file=True)
-
-      # Install the app if it does not exist.
-      android.device.install_application_if_needed(absolute_file_path,
-                                                   build_update)
+    if environment.platform() != 'ANDROID':
       return
 
-    if not build_update:
-      return
+    # Android specific initialization.
+    # Prepare device for app install.
+    android.device.initialize_device()
 
-    # The following hacks are only applicable in Chromium.
-    if utils.is_chromium():
-      return
+    # On Android, we may need to write a command line file. We do this in
+    # advance so that we do not have to write this to the device multiple
+    # times.
+    # TODO(mbarbella): Build code should not depend on fuzzing.
```

Comment:
Instead of TODO, please just move this to end of android.device.initialize_device in a helper like setup_command_line_file or something.

Reply:
Done.

Reply:
Nope, this creates a circular dependency :(

```
ImportError: Failed to import test module: tests.core.build_management.build_manager_test
Traceback (most recent call last):
  File "/usr/lib/python2.7/unittest/loader.py", line 254, in _find_tests
    module = self._get_module_from_name(name)
  File "/usr/lib/python2.7/unittest/loader.py", line 232, in _get_module_from_name
    __import__(name)
  File "/usr/local/google/home/mmoroz/projects/google/clusterfuzz/src/python/tests/core/build_management/build_manager_test.py", line 31, in <module>
    from bot.tasks import fuzz_task
  File "/usr/local/google/home/mmoroz/projects/google/clusterfuzz/src/python/bot/tasks/fuzz_task.py", line 33, in <module>
    from bot.fuzzers import builtin
  File "/usr/local/google/home/mmoroz/projects/google/clusterfuzz/src/python/bot/fuzzers/builtin.py", line 26, in <module>
    from fuzzing import testcase_manager
  File "/usr/local/google/home/mmoroz/projects/google/clusterfuzz/src/python/fuzzing/testcase_manager.py", line 33, in <module>
    from platforms import android
  File "/usr/local/google/home/mmoroz/projects/google/clusterfuzz/src/python/platforms/android/__init__.py", line 21, in <module>
    from . import device
  File "/usr/local/google/home/mmoroz/projects/google/clusterfuzz/src/python/platforms/android/device.py", line 34, in <module>
    from fuzzing import testcase_manager
ImportError: cannot import name testcase_manager
```

//cc @mbarbella-chromium, reverting this part for now